Storage/SQL/BrowserDB.swift

```diff
@@ -408,12 +408,12 @@ extension BrowserDB {
 }
 
 extension SQLiteDBConnection {
-    func tablesExist(names: [String]) -> Bool {
+    func tablesExist(names: Args) -> Bool {
         let count = names.count
-        let orClause = Array(count: count, repeatedValue: "name = ?").joinWithSeparator(" OR ")
-        let tablesSQL = "SELECT name FROM sqlite_master WHERE type = 'table' AND (\(orClause))"
+        let inClause = BrowserDB.varlist(names.count)
+        let tablesSQL = "SELECT name FROM sqlite_master WHERE type = 'table' AND name IN (\(inClause))"
```

Comment:
Nit: Looks like BrowserDB.varlist returns (?, ?, ..., ?) and we're wrapping that with more parens so the query looks like:
SELECT name FROM sqlite_master WHERE type = 'table' AND name IN ((?, ?, ..., ?))

I think we can remove the additional parens in this sql.